src/parkapi_sources/converters/opendata_swiss/models.py

```diff
@@ -0,0 +1,200 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from collections import defaultdict
+from datetime import datetime, timezone
+from enum import Enum
+from typing import Optional
+
+from validataclass.dataclasses import validataclass
+from validataclass.exceptions import ValidationError
+from validataclass.validators import DataclassValidator, EnumValidator, IntegerValidator, ListValidator, Noneable, StringValidator
+
+from parkapi_sources.converters.base_converter.pull import GeojsonFeatureGeometryInput
+from parkapi_sources.models import RealtimeParkingSiteInput, StaticParkingSiteInput
+from parkapi_sources.models.enums import ParkAndRideType, ParkingSiteType, PurposeType
+
+
+@validataclass
+class OpenDataSwissAddressInput:
+    addressLine: Optional[str] = Noneable(StringValidator())
+    city: Optional[str] = Noneable(StringValidator())
+    postalCode: Optional[str] = Noneable(StringValidator())
+
+
+class OpenDataSwissCapacityCategoryTypeInput(Enum):
+    STANDARD = 'STANDARD'
+    DISABLED_PARKING_SPACE = 'DISABLED_PARKING_SPACE'
+    RESERVABLE_PARKING_SPACE = 'RESERVABLE_PARKING_SPACE'
+    WITH_CHARGING_STATION = 'WITH_CHARGING_STATION'
+
+
+@validataclass
+class OpenDataSwissCapacitiesInput:
+    categoryType: OpenDataSwissCapacityCategoryTypeInput = EnumValidator(OpenDataSwissCapacityCategoryTypeInput)
+    total: int = IntegerValidator()
+
+
+@validataclass
+class OpenDataSwissAdditionalInformationInput:
+    de: Optional[str] = Noneable(StringValidator(multiline=False, unsafe=True))
```

Comment:
Please make sure that unsafe strings are handled later, so that they are safe in the end (most likely with an replacing regexp). Otherwise, you will end up in validation errors if you integrate these strings in ParkAPI.